htdocs/index.php

```diff
@@ -155,14 +155,14 @@
 	require_once DOL_DOCUMENT_ROOT.'/core/class/workboardresponse.class.php';
 
 	// Number of actions to do (late)
-	if (isModEnabled('agenda') && empty($conf->global->MAIN_DISABLE_BLOCK_AGENDA) && $user->rights->agenda->myactions->read) {
+	if (isModEnabled('agenda') && empty($conf->global->MAIN_DISABLE_BLOCK_AGENDA) && $user->hasRight('agenda', 'myactions', 'READ')) {
```

Comment:
If you enter 'READ', thest will be done on 
$conf->global->agenda->myactions->READ.
Not sure it works, it is better to keep the case sensitive so use 'read'.

Reply:
@eldy it's done